spec/models/manageiq/providers/hawkular/middleware_manager/middleware_server_spec.rb

```diff
@@ -0,0 +1,64 @@
+describe ManageIQ::Providers::Hawkular::MiddlewareManager::MiddlewareServer do
+  let(:ems_hawkular) do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
```

Comment:
I'm not sure of the need for this, but most specs that need a server/zone combo, can just do the following:

```ruby
_guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
...
FactoryGirl.create(:ems_hawkular, ... , :zone => zone)
```